servers/minedestiny/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "minedestiny",
+    "name": "MineDestiny",
+    "addresses": [
+        "minedestiny.hu"
+    ],
+    "primaryAddress": "play.minedestiny.hu",
+    "minecraftVersions": [
+        "1.16",
+        "1.17",
+        "1.18"
+    ],
+    "primaryMinecraftVersion": "1.18",
+    "primaryColor": "#dd544e",
+    "secondaryColor": "#a5366f",
+    "primaryLanguage": "hu",
+    "languages": [
+        "hu"
```

Comment:
This is not a supported language the only languages that are supported are:

- en
- es-EN
- it
- tr
- ar
- zh-CN
- nl
- fr
- de
- el
- ja
- ko
- lt
- no
- pl
- pl-PT
- pt-BR
- sv-SE